Form/Form.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using DUCK.Form.Fields;
+using DUCK.Utils;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace DUCK.Form
+{
+	public class Form : MonoBehaviour
```

Comment:
Class name same with namespace name. May generate chaos but we will see...

Anyway some comments would help.

Reply:
maybe namespace should be `Forms`